docs/DefaultApi.md

```diff
@@ -3595,6 +3593,7 @@ except ApiException as e:
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
  **file** | **file**| file | 
+ **skip_verify** | [**object**](.md)| If true will skip pack signature verification. Use when uploading a custom or | [optional] 
```

Comment:
Not sure what happened here, but the link is dead. Not sure where it should point, but since it's a boolean, lets remove the link and just state that it's a bool.

Reply:
Done